core/metadata.js

```diff
@@ -456,3 +457,27 @@ function buildFile (filePath /*: string */, stats /*: fs.Stats */, md5sum /*: st
   if ((stats.mode & EXECUTABLE_MASK) !== 0) { doc.executable = true }
   return doc
 }
+
+const CONFLICT_REGEXP = new RegExp('-conflict-\\d{4}(?:-\\d{2}){2}T(?:\\d{2}_?){3}.\\d{3}Z')
```

Comment:
```js
const CONFLICT_PATTERN = '-conflict-\\d{4}(?:-\\d{2}){2}T(?:\\d{2}_?){3}.\\d{3}Z'
```
Wdyt?
From what I can see the current RegExp object is not exported: we only use it 3 lines below to get its `.source` pattern... So maybe better make it a plain string and rename it to make the type clear?